frameworks/digital-outcomes-and-specialists/questions/brief-responses/dayRate.yml

```diff
@@ -14,7 +14,7 @@ validations:
     message: 'You need to answer this question.'
   - name: not_money_format
     field: dayRate
-    message: "Day rate must be in numbers only, no symbols, eg 99.95"
+    message: "Day rate must be in numbers and not include symbols, for example 99.95."
```

Comment:
The "for example" feels weird - I'm expecting an example of the symbols I shouldn't be using rather than an example of how it should look.